### ISSUE TITLE
Find-TeamsResourceAccount shows empty PhoneNumber

### ISSUE THREAD

**Issue by mwilens** [DEberhardt/TeamsFunctions#101]
Find-TeamsResourceAccount shows an empty PhoneNumber.
Please fix this or omit it.

**Comment by DEberhardt** [comment]
oh! - I just saw this after publishing v21.11 - Will address immediately
The AzureAdUser-Object does not have a PhoneNumber parameter.

**Comment by DEberhardt** [comment]
```powershell
❯ Find-TeamsRA TestAAAccount5@arkadinplatform.onmicrosoft.com


ObjectId          : 798d2d46-aea4-47ef-74da-bf50613ab8cb
UserPrincipalName : TestAAAccount5@XXXXXXX.onmicrosoft.com
DisplayName       : My TestAA Account5
UsageLocation     : GB
PhoneNumber       : tel:+13212321325
```
Fixed, live with v21.12 in a few weeks

A workaround is to pipe this one directly to the GET-Command. Hope that helps

Thank you!

**Comment by DEberhardt** [comment]
v21.12 is published - `Find-TeamsResourceAccount` now shows the correct phone number.
